platform/logging/layouts/Layouts.ts

```diff
@@ -0,0 +1,26 @@
+import { Schema } from '../../types';
```

Comment:
All this kind of generic stuff around single `PatternLayout` may seem silly, but it is just preparation for the second `JsonLayout` PR :)